roles/syslog/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+  - import_tasks: syslog-extern.yml
```

Comment:
I think this file could get a better name, so that you know what will be done in this task.